paystack/build.gradle

```diff
@@ -44,6 +44,7 @@ android {
 
         buildConfigField 'int', 'VERSION_CODE', "${rootProject.ext.versionCode}"
         buildConfigField 'String', 'VERSION_NAME', "\"${rootProject.ext.versionName}\""
+        buildConfigField 'String', 'PUSHER_KEY', '"8e4b9b7ca3418bd5cdc8"'
```

Comment:
is it intentional to be commiting a secret key to source control here?

Reply:
Yeah please can we rewrite the git history for this to not show as part of any commit and rather add it to github secrets. Try access this at compile time.